requirements.in

```diff
@@ -1,6 +1,6 @@
--e git+https://github.com/City-of-Helsinki/wagtail-svgmap.git@v0.1.2#egg=wagtail_svgmap
--e git+https://github.com/City-of-Helsinki/wagtailtinymce.git@extensibility#egg=wagtailtinymce==4.4.3.0
--e git+https://github.com/City-of-Helsinki/django-social-widgets@master#egg=django_social_widgets
+-e git+https://github.com/tomijarvi/django-social-widgets@fix-1.9-depr#egg=django_social_widgets
```

Comment:
I suppose we'll change these after deployment on server

Reply:
I can do this fix as dependencies changes have now merged.